lib/stripe/subscriptions/plan.ex

```diff
@@ -138,6 +141,7 @@ defmodule Stripe.Plan do
                optional(:nickname) => String.t(),
                optional(:product) => Stripe.id() | Stripe.Product.t(),
                optional(:trial_period_days) => non_neg_integer,
+               optional(:active) => boolean,
```

Comment:
Looks like `list`s can now be filtered by `active` as well.  Mind adding it there as well as an optional attribute?